lib/diagnostic.js

```diff
@@ -6,26 +6,32 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (string) {
+  let words = string.split(' ');
+  return words.replace(/\W+/,'').toUpperCase();
+
 };
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+const uniqueWords = function (string) {
+  let words = string.split(' ');
+  return words.replace(/\W+/,'').toUpperCase();
 };
 
 // 3. Write a function that returns the count of words in a string.
 // Provid the *option* to count unique words instead of total words.
-const wordCount = function (/* */) {
-  /* your response here */
+const wordCount = function (string) {
+  let words = string.split(' ');
+  return words.length;
 };
```

Comment:
Yeah, pretty much. You don't have the *option* piece, but the logic is sound. How could we use our other functions so we aren't rewriting code?